### PR TITLE
scripts: Use Linux MUSL toolchain for building Linux Kernel.

### PR DESCRIPTION
**Pull request by mithro**
These packages are now available via conda since;
 * https://github.com/litex-hub/litex-conda/pull/75
 * https://github.com/litex-hub/litex-conda/pull/80

### CODE REVIEW

File: scripts/build-linux.sh [hunk 1]
```diff
@@ -44,8 +44,8 @@ if [ "$FIRMWARE" != "linux" ]; then
 fi
 
 # Install a toolchain with the newlib standard library
-if ! ${CPU_ARCH}-elf-newlib-gcc --version > /dev/null 2>&1; then
```
Comment (mateusz-holenko) [inline scripts/build-linux.sh]: In line 172 we use this compiler to build VexRiscv's emulator. I suggest changing it from:
```
export RISCV_BIN="${CPU_ARCH}-elf-newlib-"
```
to:
```
export RISCV_BIN="${CPU_ARCH}-linux-musl-"
```

**Comment by mithro** [comment]
@mateusz-holenko Can you check this works on hardware and in renode?

**Comment by mateusz-holenko** [comment]
I confirm that the current master works both on HW and in Renode.

**Comment by mateusz-holenko** [comment]
I tried to switch to `${CPU_ARCH}-linux-musl-` compiler to build the VexRiscv's emulator, but it seems that it references `__libc_init_array` function in `start.S` file and it's not provided by the musl compiler.

In this context my comment from the review is invalid.

**Pull request merged**